disco_aws_automation/disco_deploy.py

```diff
@@ -625,9 +625,9 @@ def _generate_post_deploy_pipeline(self, pipeline_dict, old_group, ami):
         # If there is an already existing ASG, use its sizing. Otherwise, use the pipeline's sizing or a
         # reasonable default.
         if old_group:
-            desired_size = old_group.desired_capacity
-            max_size = old_group.max_size
-            min_size = old_group.min_size
+            desired_size = old_group.get('desired_capacity') or old_group['capacity']['target']
+            max_size = old_group.get('max_size') or old_group['capacity']['maximum']
```

Comment:
A bit weird to access the dict in two different ways in one line. Are you trying to take advantage of the fact that `old_group.get('max_size')` will return `None` if the key doesn't exist and `old_group['max_size']` will throw an exception?